### PR TITLE
add DocumentTests

### PR DESCRIPTION
**Pull request by did7121**
I changed Test method the version 2.0 to 3.0.

### CODE REVIEW

File: test/Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Tests/DocumentTests.cs [hunk 1]
```diff
@@ -526,5 +526,34 @@ public async Task Given_ServerDetails_WithEmptyRoutePrefix_When_RenderAsync_Invo
             ((string)json?.basePath).Should().BeEquivalentTo(null);
             ((string)json?.schemes[0]).Should().BeEquivalentTo(scheme);
         }
+
+        [TestMethod]
```
Comment (level120) [inline test/Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Tests/DocumentTests.cs:530]: Can you change this `TestMethod` to `DataTestMethod`?

File: test/Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Tests/DocumentTests.cs [hunk 2]
```diff
@@ -526,5 +526,34 @@ public async Task Given_ServerDetails_WithEmptyRoutePrefix_When_RenderAsync_Invo
...
+        public async Task Given_ServerDetails_WithEmptyRoutePrefix_When_RenderAsync_Invoked_Then_It_Should()
```
Comment (level120) [inline test/Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Tests/DocumentTests.cs:531]: If you change to `DataTestMethod` you may like to add `DataRow(...)` as well

**Review by level120 — CHANGES_REQUESTED** [review]
Thanks for your PR! Would you please take further look with comments below?

**Review by justinyoo — APPROVED** [review]
LGTM! Thanks for your contribution!

**Pull request merged**